api/src/controllers/website.js

```diff
@@ -81,45 +81,18 @@ async function deleteWebPage(req, res, next) {
  * @returns 200: publication options successfully updated in the DB
  * @returns 500: Server error while saving new page name to DB
  */
-async function updatePublicationOptions(req, res, next) { // eslint-disable-line no-unused-vars
-  const updatedPubOptions = req.body;
-  const { teamId: _id } = req.params;// TODO: teamId get from the token instead of parameters?
-  try {
-    await Website.updateOne(
-      { teamId: _id },
-      {
-        $set: {
-          publicationOptions: updatedPubOptions,
-        },
-      },
-    );
-    return res.status(200).json(updatedPubOptions);
-  } catch (err) {
-    return res.send(fillErrorObject(500, 'Server error', [err.errors]));
-  }
-}
+async function patchClientWebMetadata(req, res, next) { // eslint-disable-line no-unused-vars
```

Comment:
To clarify, I decided to use a patch endpoint so we don't need to have operate endpoints (and thus functions) for updating publication options, title (if we go ahead with it) and layout and theme.

Reply:
Could we call this `updateClientWebMetadata` instead? Would like the naming to be consistent with the frontend and prefer 'update' over 'patch'.